### ISSUE TITLE
WCRP - Improvement of Attendance Policies and Clarification of Sanctions for WCA Sanctioned Competitions and Events.

### ISSUE THREAD

**Issue by Samuel-Baird** [thewca/wca-documents#338]
> 4. Be open to **all** competitors and spectators who wish to attend, up to the maximum capacity
stated on the website or otherwise permitted by the venue.

Currently, the WCRP allows _anyone_ to attend competitions. It may be beneficial to create a subclause to clarify that competitors who are banned from attendance of competition cannot attend to align with point 1.1 of [Suspensions and Sanctions](https://www.worldcubeassociation.org/documents/motions/15.2022.1%20-%20Suspensions%20and%20other%20Sanctions.pdf)). The Suspensions and Sanctions document currently only applies to Registered Speedcubers but the WCA Board is currently evaluating a proposal to change the document to allow these same sanctions to be applied to Competition Organizers, Guests, and WCA Officials in addition to Registered Speedcubers. As a result, it may be beneficial to not specify "Registered Speedcubers" or "competitors", but rather use a neutral word like "individuals" that can apply to anyone. 

>Example:
>4.2 Individuals suspended from attending WCA sanctioned competitions may not attend any WCA sanctioned competitions until the period of their suspension has expired (see 1.1 of [Suspensions and other Sanctions](https://www.worldcubeassociation.org/documents/motions/15.2022.1%20-%20Suspensions%20and%20other%20Sanctions.pdf)). 

Additionally, there may be instances where organizers/delegates may not be able to make contact with the WDC in order to remove individuals who may engage in disruptive actions or pose a threat to the safety or well-being of anyone in attendance of a WCA sanctioned competition. [11h](https://www.worldcubeassociation.org/regulations/#11h) of the WCA Regulations covers this well for competitors but fails to include non-competitors. I believe that a clause with similar wording could cover this neatly. 

> Example:
> 4.3 A WCA Delegate may require individuals to leave the competition venue for behavior that is unlawful/violent/indecent/unsafe, or intentionally damages venue facilities or personal property within the venue. Delegates should take into account the seriousness of the situation and the best possible course of action. Delegates should also report any such incidences to the WDC to investigate the need for disciplinary action.

if there are concerns for abuse then this could be modified to something like “all cases must be reported to the WDC for review”

**Comment by sebastianotronto** [comment]
(Note: the WDC has not discussed this internally yet, but there is no good reson why this discussion should happen privately instead of benefitting from the larger WCA Staff and community).

I do not have much to say about the first proposal, as it is mostly a matter of consistency and improved clarity.

I personally agree with the second proposal. In case of serious behavioral issues the local delegates are often the best equipped to make a quick decision, and it is inconvenient that they have to wait for the WDC to carry out an investigation just to keep away dangerous individuals. In this kind of situation it makes more sense that the WDC simply review the case, approving or rejecting the delegates' decision. We can also leverage our external point of view and act as a board of appeal.

I would also like to clarify that I oppose extending this to cheating incident. For that, [regulation 2j](https://www.worldcubeassociation.org/regulations/#2j) is already enough.

**Comment by EdHollingdale** [comment]
Good suggestion Samuel - I definitely agree with the first addition. 

For the second, I think it is worth adjusting this to focus more on regional organisations. The WCA is not an organiser of competitions, and I think it is important there is a clear distinction between the roles WCA officials play (ensuring the competition runs according to the regulations and policies) and the role of organisers (who put on the event). To me a person who is behaving in an unsafe manner is an organiser problem not a WCA problem. Organisers should be the ones managing this, not WCA Delegates.

It is worth noting that a lot of regions in the world are moving away from the "local Delegate" being the leader of the given community, and having clear Regional Organisations (noting that many of the key people in these organisations are also WCA Delegates). These Regional Organisations should be empowered by WCA policies to manage problematic individuals for cases where it isn't the responsibility of the WCA.

**Comment by Samuel-Baird** [comment]
Hey Ed,

While I do believe that having regional organizations handle situations like this in regions like Australia, I don't believe that it would be able to function nearly as well in other regions such as the USA. CubingUSA for example is involved in very few competitions and even the ones it is involved in it is oftentimes are only supported financially (ie there isn't a representative of CubingUSA at the competition). The person handling the situation should be at the competition and currently, the only person who is actually required to be there by the WCA policies and Regulations is a WCA delegate, while RO representatives are not (even organizers can skip the actual competition itself if they contribute significantly to the pre-work). I would also have a concern about regions where no RO is present. I also agree that the organizer would be the ideal person to handle this but I know that a lot of our organizers throughout the world are fairly young and lack the maturity and experience to handle these situations, while on the other hand, I would say that most Delegates tend to be older and more mature. 

I have seen a lot of great recommendations surrounding ROs that would probably provide some structure that would be needed for a change like this to be run well by ROs. However, those changes will likely take time and we recently experienced several incidents already where delegates/organizers have felt powerless in being able to handle these situations. As a result, I currently would currently prefer to prioritize competitor safety by working on giving Delegates the power to do this ASAP. After we have that change in effect so that situations could be handled, I would love to be able to take some more time to carefully look into how we could transition over to ROs handling the situation. 

TLDR; delegates are always at competitions, meaning this change would be a quick one size fits all approach that is needed given our growing incident rates. Looking into how the responsibility could be transferred to ROs after the "quick fix" has been put into effect, as it may take more time/changes

**Comment by EdHollingdale** [comment]
I'd be okay if there is some power given to delegates here - as I implied there are still a lot of regions that still operate under the 'local delegate' style model.

But I don't accept that because some parts of the world are not as organised/developed as others from a cubing organisation sense that the WCA shouldn't be starting the move in the direction to support places like the UK and Australia. In the US specifically there are a lot of sub-regions with pretty good organisations, they should be recognised and empowered to (so I think your points on CubingUSA aren't applicable). 

Introducing it as a WCA Delegate decision opens up the WCA to additional liability however. Often problematic individuals are the type who might threaten to sue. Some delegates might see this policy and think they can enforce it when they are actually in an area of the world where it is unenforceable (which is why regional organisations are much better placed to navigate these local level nuances, the WCA policy should just enable them to have such restrictions where it is possible and they are willing to take on any associated legal liability).

**Comment by Samuel-Baird** [comment]
Just to make sure we are on the same page, are you envisioning that ROs handle it when possible but in regions where ROs do not exist it would become a responsibility for the Delegate(s) that cover that region? Like some sort of "hybrid system" where ideally it is handled by an RO but it can be passed off to others when ROs are unavailable. 

I was thinking that you were referring to officially recognized ROs, if "sub-ROs" were to handle these sorts of incidents would the WCA or official ROs first need to create some sort of system to recognize sub-ROs, or were you thinking that unofficial ROs should be able handle these kinds of incidents?

The liability issue is a good point, but I'm not sure I am following what you mean about how some delegates may not be able to enforce the policy in an area where it is not enforceable, could you give an example for how attempting to enforce one of these proposed policies may not be doable?

**Comment by EdHollingdale** [comment]
> are you envisioning that ROs handle it when possible but in regions where ROs do not exist it would become a responsibility for the Delegate(s) that cover that region?

Yes, notwithstanding the issues around liability

> if "sub-ROs" were to handle these sorts of incidents would the WCA or official ROs first need to create some sort of system to recognize sub-RO

Yes, this is something that should be done for the US. In effect the WCA does already 'recognise' many of these sub-ROs if they have set up dedicated invoicing with the WFC. 

> could you give an example for how attempting to enforce one of these proposed policies may not be doable?

At many venues in Australia it may not be legally possible to enforce a "we don't like you for X reason, please leave" policy at an 'open' event, we would need to wait for something bad to happen (can't do it based on suspicion / incidents which weren't then handled by the correct authorities). I imagine it is similar in a lot of places where there are laws to protect people against discrimination.

**Comment by Samuel-Baird** [comment]
> > are you envisioning that ROs handle it when possible but in regions where ROs do not exist it would become a responsibility for the Delegate(s) that cover that region?
> 
> Yes, notwithstanding the issues around liability
> 
> > if "sub-ROs" were to handle these sorts of incidents would the WCA or official ROs first need to create some sort of system to recognize sub-RO
> 
> Yes, this is something that should be done for the US. In effect the WCA does already 'recognise' many of these sub-ROs if they have set up dedicated invoicing with the WFC.
> 
> > could you give an example for how attempting to enforce one of these proposed policies may not be doable?
> 
> At many venues in Australia it may not be legally possible to enforce a "we don't like you for X reason, please leave" policy at an 'open' event, we would need to wait for something bad to happen (can't do it based on suspicion / incidents which weren't then handled by the correct authorities). I imagine it is similar in a lot of places where there are laws to protect people against discrimination.

I support anything that allows this to be fixed quickly. If needing to create a system to officially recognize sub ROs is needed before this could go into effect then I would strongly prefer updating the WCRP as quickly as possible to allow delegates to handle these situations for the time being, as I have seen how long even simple changes to the WCA can take. Of course, the eventual plan to expand/transition this over to ROs would be ideal, but I would rather address this issue quickly than significantly delay something that poses direct threats to safety as opposed to creating significant delays to it the "optimal way" all in one update.

**Comment by Epride** [comment]
Just chiming in to note that a system to officially recognise sub-RO's already exists in motion 6.2022.1 art 1.1 "1.2 Regional Organizations may recognize a separate organization to act on its behalf in a subsection of the region of which it covers. This organization shall have the full rights, duties and responsibilities of an Associate Regional Organization."

**Comment by EdHollingdale** [comment]
Hi Samuel, I think amending your wording to also give the authority to ROs does offer the quick fix you're looking for though. I've acknowledged that there are many regions which don't have ROs and where WCA Delegates will still need to be empowered, per your wording. But we shouldnt let that stop the WCA from giving ROs that are organised the ability to handle incidents like this instead of WCA Delegates. 

TBH I think this should be exclusively the domain of ROs and not give potential liability to the WCA by empowering WCA Delegates to handle it. But given how few regions are well enough organised to handle this the Board can make a decision that given the setupsness they will give WCA Delegates the ability to handle these things (in addition to ROs).

TLDR: I'm not saying we need an intricate system to recognise all cubing organisations. We just need to make sure we give the power to handle issues like this to ROs in addition to (or instead of) WCA Delegates.

Per Ethan's comment, the tools for recognising sub-ROs exist. All the better if CubingUSA can get organised to officially recognise the sub-ROs operating in the US so that they can also be covered.

**Comment by slongbehn** [comment]
Jumping in here into the discussion. 

> I think this should be exclusively the domain of ROs and not give potential liability to the WCA by empowering WCA Delegates to handle it.

I am on board with this and many of what was said, but I am not officially a part of an RO, but they sponsor our competitions. Is there clarity on whether I can then ask someone problematic to leave? Do I have to have someone who officially represents that RO to do it?

**Comment by Samuel-Baird** [comment]
@Epride thanks for pointing that out! I was unaware that sub-ROs are already defined by the WCA (I am unaware of any current operating ones) so knowing that the framework is already there helps this out significantly :)

@EdHollingdale I like the idea of defaulting to ROs to handle this but then allowing Delegates to step in when ROs are not available. What are your thoughts on something like the wording below? I'm not sure about the exact wording, but I think it captures the the Spirit of the ideas that have been discussed here.

>4.3 Individuals may be required to leave the competition venue due to unlawful, violent, indecent, or unsafe behavior. Additionally, intentional damage to venue facilities or personal property within the venue may also warrant removal.
>4.3.1 The responsibility for handling all instances requiring someone to be removed from a competition venue lies with the Regional Organization. This can be carried out by a director or officer of the Regional Organization, or by an individual appointed by the regional organization to act on its behalf in such incidents.
>4.3.2 In regions where a Regional Organization is not present, these incidents may be handled by a WCA Delegate.
>4.3.3 The individual handling the incident should consider the severity of the situation and determine the most appropriate course of action, including contacting local legal authorities if necessary. Delegates should also promptly report such incidents to the WDC for investigation and potential disciplinary action.

**Comment by zekemackay06** [comment]
Implemented in #384
